quipucords/quipucords/urls.py

```diff
@@ -17,38 +17,11 @@
 
 from django.conf.urls import include
 from django.contrib import admin
-from django.contrib.auth import views as auth_views
-from django.urls import path, re_path
+from django.urls import path
 from django.views.generic import RedirectView
-from django.views.generic.base import TemplateView
 
 urlpatterns = [
-    path("login/", auth_views.LoginView.as_view(), name="login"),
-    path("logout/", auth_views.LogoutView.as_view(), name="logout"),
     path("admin/", admin.site.urls),
     path("api/", include("api.urls")),
-    path("", RedirectView.as_view(url="/login", permanent=False), name="home"),
-    # ui routing
-    re_path(
-        r"^(client/(sources|scans|credentials|)(/|)(index.html|))$",
-        TemplateView.as_view(template_name="client/index.html"),
-        name="client",
-    ),
-    # docs files
-    re_path(
-        r"^client/docs(/|)(index.html|use.html|)$",
-        RedirectView.as_view(url="/client/docs/use.html", permanent=False),
-        name="docs",
-    ),
-    re_path(
-        r"^client/docs(/|)(install.html|)$",
-        RedirectView.as_view(url="/client/docs/install.html", permanent=False),
-        name="docs",
-    ),
-    # static files (*.css, *.js, *.jpg etc.)
-    re_path(
-        r"^(?!/?client/)(?P<path>.*\..*)$",
-        RedirectView.as_view(url="/client/%(path)s", permanent=False),
-        name="client",
-    ),
+    path("", RedirectView.as_view(url="/admin/login", permanent=False), name="home"),
```

Comment:
now that we don't have the template for login, the auth view for login won't work.

I decided to point home to admin just so we can use it to login for local development.
I know we don't actually have the intention of keeping admin (having it is an accident AFAIK), but not having a session login makes manually test the api a bit harder.

Quick alternatives I could think right now:
- I could enable django admin and views conditionally, only on debug mode
- Could add a simple replacement for login view (and have that view only available on debug mode)

Reply:
My ranked votes are 1) enable admin/views and this `path` line only if debug is True, or 2) just axe the admin and this path entirely (does anybody use admin?)